examples/http/opnsense_interfaces_ipv6.js

```diff
@@ -0,0 +1,136 @@
+/**
+ * Domotz Custom Driver 
+ * Name: OPNSense Interfaces Stats IPV6
+ * Description: This script is designed for retrieving IPv6 interface statistics from an OPNsense firewall.
+ * 
+ * Communication protocol is HTTPS
+ * 
+ * Tested with FreeBSD OPNsense version: 13.2-RELEASE-p5
+ *
+ * Creates a Custom Driver Table with the following columns:
+ *      - Cleared: The timestamp when the statistics were last cleared or reset
+ *      - References: The number of references or connections associated with the interface
+ *      - In traffic (ipv6 block): The amount of incoming IPv6 traffic that was blocked by the firewall
+ *      - In traffic (ipv6 pass): The amount of incoming IPv6 traffic that was allowed or passed by the firewall
+ *      - Out traffic (ipv6 block): The amount of outgoing IPv6 traffic that was blocked by the firewall
+ *      - Out traffic (ipv6 pass): The amount of outgoing IPv6 traffic that was allowed or passed by the firewall
+ * 
+ **/
+
+// interfaceName: Set it to 'ALL' to retrieve all interfaces,
+// or specify a list of interfaces to filter and display only the selected interfaces.
+var interfaceName = D.getParameter("interfaceName");
+
+// Custom Driver Table to store IPv6 Interface Statistics
+var table = D.createTable(
+    "Interfaces Advanced IPV6",
+    [
+        { label: "Cleared", type: D.valueType.DATETIME },
```

Comment:
It should be valueType not type

Change "Cleared" to "Cleared Time" (same as IPv4 driver)
Change "References" to "References Number" (same as IPv4 driver)